config.md

```diff
@@ -47,6 +47,7 @@ The parameters are similar to the ones in [the Linux mount system call](http://m
 
 * **`destination`** (string, REQUIRED) Destination of mount point: path inside container.
   For the Windows operating system, one mount destination MUST NOT be nested within another mount (e.g., c:\\foo and c:\\foo\\bar).
+  The destination MUST be present in the root filesystem. Runtime MUST NOT create the destination.
```

Comment:
Why can't we just make this a restriction which only applies if `readonly` is set to true for the `rootfs`. I get why you'd want a sweeping statement like this, but as @wking mentioned it does make nested `tmpfs` mounts quite hard -- and in fact would make runC's method of handling cgroups not really possible anymore.